src/functions-and-arrays.js

```diff
@@ -1,19 +1,77 @@
 // Iteration #1: Find the maximum
 
+function maxOfTwoNumbers(a,b) {
+  if (a<b)  {
+    return b;
+  } if (a>b) {
+    return a
+  } if (a==b) {
+    return a;
+  }
+}
+  
+
 // Iteration #2: Find longest word
 const words = ['mystery', 'brother', 'aviator', 'crocodile', 'pearl', 'orchard', 'crackpot'];
+let wholeArr=[];
+let longest;
+
+function findLongestWord(words) {
+  var tlength = 0;
+  for(var i =0; i < words.length; i++){
+    if(tlength < words[i].length){
+      tlength = words[i].length;
+    }
+  }
+  for(var j =0; j < words.length; j++){
+    if(words[j].length == tlength){
+       wholeArr.push(words[j]);
+    }
+  }
+ if(wholeArr.length == 1){
+   return wholeArr[0]
+ }else{
+    return wholeArr
+}
+}
```

Comment:
the logic that you used is on the right way, but it is a bit simpler than what you tried.
here is one solution for this interaction
```
let longestWord = "";
  for (let i = 0; i < words.length; i++) {
    if (words[i].length > longestWord.length) {
      longestWord = words[i];
    }
  }
  return longestWord;
```